src/data/showcases.js

```diff
@@ -1193,6 +1193,15 @@ export const Showcases = [
     source: null,
     tags: ["daotool", "governance"],
   },
+  {
+    title: "Orcfax",
+    description:
+      "Orcfax is a decentralized oracle service designed to publish data about real world events to the Cardano blockchain. Orcfax data is made available to on-chain smart contracts in Cardano's eUTXO native format using the Orcfax Protocol.",
```

Comment:
@ross-spencer - given that this `description:` is focussed on the service, why is this submitted in the Showcase and not as a Builder Tool? = https://developers.cardano.org/tools/?tags=oracle

Reply:
Showcase also has the oracle category. I believe that it would be the right place: https://developers.cardano.org/showcase?tags=oracle

Reply:
Resolving since this has been added to the `oracle` category in Builder Tools instead.